README.md

```diff
@@ -135,21 +135,46 @@ logging_inputs:
     type: basics
 ```
 
-3) Deploying basic LSR/Logging config files in /etc/rsyslog.d, which handle inputs from the local system and remote rsyslog and outputs into the local files.
+3) Deploying basic LSR/Logging config files in /etc/rsyslog.d, which handle inputs from the remote rsyslog and outputs into the per host base local files.
 ```
-logging_enabled: true
-rsyslog_capabilities: [ 'network', 'remote-files' ]
-logging_purge_confs: true
-logging_outputs:
-  - name: local-files
-    type: files
-logging_inputs:
-  - name: system-and-remote-input
-    type: basics
-logging_flows:
-  - name: flow0
-    inputs: [system-and-remote-input]
-    outputs: [local-files]
+        logging_outputs:
+          - name: default_remote_files
```

Comment:
Why is it called `default_remote_files`? Is it a special name or just an arbitrary name that you introduced in the example? If it is just an arbitrary name, why is it called `default_something`? To me `default` sounds like something predefined.

Reply:
> Why is it called `default_remote_files`? Is it a special name or just an arbitrary name that you introduced in the example? If it is just an arbitrary name, why is it called `default_something`? To me `default` sounds like something predefined.

The name can be any string, but since there is no `remote_log_path` nor `remote_sub_path` is defined in the remote_files output, the [default configuration](https://github.com/linux-system-roles/logging/pull/116/files#diff-b23a18f65552498ad31bf927da164818R20-R55) is used.

Reply:
Replaced `default_remote_files` with `remote_files_output`.